src/aal/aal.h

```diff
@@ -11,6 +12,10 @@
 
 #if defined(__arm__) || defined(__aarch64__)
 #  define PLATFORM_IS_ARM
+   /**
+    * on ARM cpu counters are accessible only in privileged mode
+    */
+#  define SNMALLOC_NO_ALL_CPUCOUNTERS
```

Comment:
Can this be an AAL feature flag?  I'd rather minimise the pollution to the global namespace where possible.